### PR TITLE
ODIM-5698 Limiting the resources which are retrieving from BMC

### PR DESCRIPTION
**Pull request by irappa-pattar**

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

File: lib-utilities/response/errorresponse.go [hunk 1]
```diff
@@ -312,6 +312,15 @@ func (a *Args) CreateGenericErrorResponse() CommonError {
 					Severity:   "Warning",
 					Resolution: "Add properties in the JSON object and resubmit the request.",
 				})
+		case RateLimitExceded:
```
Comment (amar-shalgar) [inline lib-utilities/response/errorresponse.go]: Correct the spelling

File: lib-utilities/response/response.go [hunk 2]
```diff
@@ -79,6 +79,8 @@ const (
 	PropertyValueConflict = BaseVersion + "PropertyValueConflict"
 	// NoOperation  defines the status message at the time of of there is no opeartion need to be performed.
 	NoOperation = BaseVersion + "NoOperation"
+	// RateLimitExceded  defines exceded the number of requests/resources.
+	RateLimitExceded = BaseVersion + "GeneralError"
```
Comment (amar-shalgar) [inline lib-utilities/response/response.go]: correct the spelling for RateLimitExceded

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

File: lib-utilities/response/errorresponse.go (hunk 1 above)
Comment (irappa-pattar) [reply lib-utilities/response/errorresponse.go]: corrected

File: lib-utilities/response/response.go (hunk 2 above)
Comment (irappa-pattar) [reply lib-utilities/response/response.go]: corrected

**Review by amar-shalgar — APPROVED** [review]
LGTM

**Pull request merged**